### PR TITLE
Updated to ESLint

### PR DESCRIPTION
**Pull request by nagu165**
I modified the ESLint package and also made some minor adjustments to index.js file.
Thank you.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var crypto = require('crypto');
+const crypto = require('crypto');
```
Comment (ljharb) [inline index.js:3]: this is a breaking change for pre-const engines.

**Review by ljharb — CHANGES_REQUESTED** [review]
Why? I prefer the eslintrc format; nothing needs updating.

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
 		"@ljharb/eslint-config": "^21.1.0",
 		"aud": "^2.0.3",
 		"auto-changelog": "^2.4.0",
-		"eslint": "=8.8.0",
+		"eslint": "^8.8.0",
```
[the remark this answers is not in the document]
Comment (nagu165) [reply package.json:46]: yes I understand the issue now . 
Thankyou for the response I will try to improve upon my shortcomings.

File: index.js (hunk 1 above)
Comment (nagu165) [reply index.js:3]: Yeah an oversight on my part.
Thanks for the response.

**Review by Abeanna — APPROVED** [review]

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Fayis4747) [reply package.json:46]: { 
         "root": true, 
  
         "extends": "@ljharb", 
  
         "rules": { 
                 "func-style": "warn", 
                 "indent": ["error", 2], 
                 "multiline-comment-style": "off", 
                 "sort-keys": "off", 
         }, 
  
         "overrides": [ 
                 { 
                         "files": "browser/index.js", 
                         "rules": { 
                                 "func-name-matching": "off", 
                                 "max-statements-per-line": "off", 
                                 "no-underscore-dangle": "warn", 
                         }, 
                 }, 
                 { 
                         "files": "browser/verify.js", 
                         "rules": { 
                                 "max-params": "off", 
                                 "max-statements": "off", 
                                 "max-statements-per-line": "off", 
                                 "no-param-reassign": "warn", 
                                 "no-plusplus": "warn", 
                                 "no-use-before-define": "warn", 
                         } 
                 }, 
                 { 
                         "files": "browser/sign.js", 
                         "rules": { 
                                 "max-params": "off", 
                                 "max-statements-per-line": "off", 
                                 "no-param-reassign": "warn", 
                                 "no-plusplus": "warn", 
                                 "no-use-before-define": "warn", 
                         } 
                 }, 
                 { 
                         "files": "test/*.js", 
                         "rules": { 
                                 "max-lines-per-function": "off", 
                         }, 
                 }, 
         ], 
 }

**Pull request closed without merging**